<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Mansoor" ignored="0" name="Chrome Test" tests="10" failures="1" timestamp="02 Jun 2019 20:18:11 GMT" time="54.379" errors="0">
  <testcase name="testCase" time="0.011" classname="FrameWork.TestNG.TestNGBasics"/>
  <testcase name="testTheLoginBtn" time="1.701" classname="FrameWork.TestNG.LoginTest"/>
  <testcase name="testLogIn" time="4.492" classname="FrameWork.TestNG.LoginTest">
    <failure type="java.lang.AssertionError" message="The app did not log in expected [My Account] but found [Account Login]">
      <![CDATA[java.lang.AssertionError: The app did not log in expected [My Account] but found [Account Login]
at FrameWork.TestNG.LoginTest.testLogIn(LoginTest.java:51)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- testLogIn -->
  <testcase name="testTheShoppingCartContinueBtn" time="3.315" classname="FrameWork.TestNG.Practice1"/>
  <testcase name="hpLaptop" time="7.956" classname="FrameWork.TestNG.Practice2"/>
  <testcase name="macBook" time="4.757" classname="FrameWork.TestNG.Practice2"/>
  <testcase name="loginWithCorrectUserAndPass" time="2.074" classname="FrameWork.TestNG.Practice1"/>
  <testcase name="loginWithIncorrectUserAndCorrectPass" time="1.938" classname="FrameWork.TestNG.Practice1"/>
  <testcase name="loginWithCorrectUserAndIncorrectPass" time="2.346" classname="FrameWork.TestNG.Practice1"/>
  <testcase name="loginWithIncorectUserAndPass" time="2.556" classname="FrameWork.TestNG.Practice1"/>
</testsuite> <!-- Chrome Test -->
